lerna.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packages": [
+    "common/predictive-text",
+    "common/lexical-model-types",
+    "web"
+  ],
+  "version": "14.0.0"
```

Comment:
Manually.  We're not publishing, so there's no auto-update of version numbers.

That said, the version number actually does matter a bit; if Lerna can't find a matching version for a package within the repo, it won't set up the internal links.  So, I boosted the packages it'll manage to 14.0.0 so that there's at least a simple baseline for this.

For the future:  `lerna publish`, should we ever start to use that to formally publish our packages, apparently can auto-submit PRs to increment version numbers.